app/uk/gov/hmrc/bindingtariffclassification/service/CaseService.scala

```diff
@@ -34,6 +37,14 @@ class CaseService @Inject()(appConfig: AppConfig,
     caseRepository.insert(c)
   }
 
+  def addInitialSampleStatusIfExists(c: Case): Future[Unit] = {
```

Comment:
Having discussed the need for the initial "Operator -1" event - only other observation is that maybe this method could be private and called from the 'insert' method above (as that is the only time 'addInitialSampleStatusIfExists' needs to be called - when creating a new case.